runtime/bcutil/ClassFileWriter.cpp

```diff
@@ -957,6 +968,9 @@ ClassFileWriter::writeAttributes()
 	if (J9_ARE_ANY_BITS_SET(_romClass->extraModifiers, J9AccRecord)) {
 		attributesCount += 1;
 	}
+	if (J9_ARE_ANY_BITS_SET(_romClass->extraModifiers, J9AccSealed)) {
```

Comment:
Please use `J9ROMCLASS_IS_SEALED` at every possible location rather than testing the bit directly.